### DISCUSSION TITLE
Getting error - ERR_INVALID_URL

### DISCUSSION THREAD

**Discussion by rohen-compatio** [mfts/papermark#361] [Q&A] [answered]
<html><head></head><body>

i'm new to next.js and trying to setup papermark on vercel. while building, i'm getting this following error. 

TypeError: Failed to parse URL from undefined/api/alternatives
--

<img width="1431" alt="image" src="https://github.com/mfts/papermark/assets/150922932/ec8eef90-31c1-4448-98af-46da1478c66c">

<span>    input: 'undefined/api/alternatives'</span></body></html>

**Comment by mfts** [comment]
Thanks for opening it. Replied to your issue.

**Comment by mfts** [comment, marked as answer]
Fixed here: https://github.com/mfts/papermark/issues/362
